### DISCUSSION TITLE
Bentley MHI2_ER_BYG24_P4220_MU1197_PATCH  how to update maps 2023/2024

### DISCUSSION THREAD

**Discussion by Mriusz69** [Mr-MIBonk/M.I.B._More-Incredible-Bash#201] [Q&A] [answered]
Hello

I have such a problem
I uploaded MIB 3.5.0, I was able to install AA / CP and update navigation maps for 2022/2023. It was the map version P197_N60S5MIBH3_EU. When I try to update the maps for 2023/2024 P330_N60S5MIBH3_EU now, the system tells me that there is no data to update.
Please help

[MU1197-JTB-07427.10.181191015A.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11413834/MU1197-JTB-07427.10.181191015A.log)

**Comment by harman-f** [comment]
>the system tells me that there is no data to update. Please help
Picture of this would be helpful.

Through which menu do you try to update?

How does the SD root looks like after you copied the files on?

**Comment by Mriusz69** [comment]
![20230507_112702](https://user-images.githubusercontent.com/131761189/236669797-065a9826-7bbd-4df0-bd4b-d0a808bee69e.jpg)
![20230507_112745](https://user-images.githubusercontent.com/131761189/236669808-3707b20c-e38d-497a-b8f3-d32f60f544a9.jpg)
![20230507_112853](https://user-images.githubusercontent.com/131761189/236669817-8e44c683-a7c9-4ca6-a685-7164e53cd131.jpg)
![20230507_112902](https://user-images.githubusercontent.com/131761189/236669821-2b99cefd-72b5-4743-9d9e-1f83bfb163ca.jpg)
![20230507_112947](https://user-images.githubusercontent.com/131761189/236669827-45be9cd5-e151-4cb8-9ea5-b1f1525f88de.jpg)

**Reply by tbilic14** [reply]
Same thing happened to me. 
use a fresh new sd card. And use windows to format it and copy the files to it. Mac doesn’t work. 
Also best to use sd card formatter utility.

**Comment by Mriusz69** [comment]
<img width="888" alt="Zrzut ekranu 2023-05-7 o 11 42 08" src="https://user-images.githubusercontent.com/131761189/236669953-839468c2-978b-4155-9822-d14bb0d646e2.png">

**Comment by Mriusz69** [comment]
I copied the same files with maps ver.330 as maps ver.197
The map wer.197 has been uploaded, and the new one does not want to

**Reply by harman-f** [reply]
Would be easier, if you could switch your unit to english for the pictures.
Looks like you are trying to update maps via map menu.

You have to update maps via engineering menu:
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/wiki/%22hidden-menus%22---key-combinations#byg24

**Comment by Mriusz69** [comment]
I tried through the engineering menu and they also do not want to update the map.
Only why maps wer.189 updated for me to ver. 197 via the navigation menu
Or maybe this is the cause:

search File MU1197-ifs-root-part2-0x00ba0000-8836F300.ifs

No Flashfiles there!

**Reply by harman-f** [reply]
Unit is already patched, that is the reason, why yout get No Flashfiles there!

Take s screenshot of the update in engineering menu

**Comment by Mriusz69** [comment]
![20230507_131436](https://user-images.githubusercontent.com/131761189/236675412-d8062f8f-3d26-492d-a363-aac69d7c1698.jpg)
the last item ENABLE UPDATE is not highlighted.
I waited 15 minutes and nothing

**Comment by Mriusz69** [comment]
I download maps from this site:
    http://map.audi-mib.ru/

**Reply by harman-f** [reply]
P330 EU maps are latest, download directly from VAG is best.

Try to format navdb and try again 
M.I.B. --> Advanced Settings --> Clean Navigation DB

**Comment by Mriusz69** [comment]
I only have option ---> format_nav_db

**Reply by Mriusz69** [reply]
After the format now I have nothing.
No navigation data.
How do I restore this data now

**Reply by Mriusz69** [reply]
After the format, I also did not want to update to ver.330

**Reply by harman-f** [reply]
There are known issues with creating SD cards on MAC. Could be related to that.

**Reply by Mriusz69** [reply]
How do I restore the navdb

**Reply by Mriusz69** [reply]
Please tell me how to restore the copy so that it is good

**Reply by harman-f** [reply]
Your navigation data on unit is deleted now. 
It has to be reinstalled from SD. 

Most likely the cause is, that you use a Mac to create the update SD. Any non Mac around to create the SD card?

Other possibilities are wrong coding or missing FECs. 

Something changed, since you installed maps last time.

**Comment by Mriusz69** [comment]
![236675412-d8062f8f-3d26-492d-a363-aac69d7c1698](https://user-images.githubusercontent.com/131761189/236701280-356bca9b-8692-4140-9794-77b3de617a9e.jpg)
how do i upload maps ver.197 .
There are no items in the navigation menu to update the maps.
The engineering menu sees the files but I wait 20 minutes and nothing happens.
The last three items are unhighlighted and I can't do anything further

**Reply by harman-f** [reply]
trying to install P197 will not change anything. 
Go for P330 and create your SD with something, which is not a MAC....

**Reply by Mriusz69** [reply, marked as answer]
!!!!!!!! NOTHING SHOULD BE DONE ON A MAC!!!!!!!!
The great truth about working on Windows.
All formatting operations, software uploading must be done on Windows.
After a whole wasted day, I managed to get everything done. I have maps V03959802TR 0330 ECE 2023 / 2024. and the system works perfectly.

**Reply by Mriusz69** [reply]
I'll post a photo of the software version tomorrow

**Reply by harman-f** [reply]
Would be great to get a copy of your units backup :)

**Reply by Mriusz69** [reply]
From the backup folder

**Reply by Mriusz69** [reply]
How do I upload 120MB

**Reply by harman-f** [reply]
filehorst.de

I would also be interested into an advanced backup :)
Will take about 45+Minutes of time

Select ALL:
![grafik](https://user-images.githubusercontent.com/98130152/236908356-1350f922-5780-4c41-9eb2-25b5038856f6.png)

**Reply by Mriusz69** [reply]
I will make a new SD card with MIB. I'll send a copy tomorrow

**Reply by Mriusz69** [reply]
I'm sorry but I waited 1.5 hours to make a copy and I stopped making a copy it took too long.
Thank you for your help

**Reply by harman-f** [reply]
That is too long. Send me what you find in backup folder. 😊

**Reply by Mriusz69** [reply]
Please let me know when you download the files.
Then I'll remove them.
Regards

**Comment by Mriusz69** [comment]


**Reply by Mriusz69** [reply]
![20230520_104756](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/131761189/d5f4af5f-9e47-433d-98e9-084d8014ee81)

**Comment by Mriusz69** [comment]
...

**Reply by harman-f** [reply]
Thanks, download is finihed ;)

**Comment by Mriusz69** [comment]
Hello
Can I upgrade GRACENOTE EU V17
I found this version of GRACENOTE2 EU V22

**Reply by harman-f** [reply]
Yes, should also work for you

**Reply by Mriusz69** [reply]
Hello
What is GRACENOTE2 AIO PAYLOAD for.

**Reply by harman-f** [reply]
Only used for AIO FWs

**Reply by Mriusz69** [reply]
![20230520_104756](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/131761189/4f9e668f-5fdc-4d2e-a4fe-d0fa5529765b)

**Comment by Mriusz69** [comment]
Hello
Is it possible to do something like this when the radio is on and music is playing, in addition to the logo of the radio station, the album cover of the band that is now playing on the radio can be displayed. I saw something like this in Lambo URUS

**Reply by harman-f** [reply]
That is what GraceNote should do.

**Reply by Mriusz69** [reply]
Hello
I updated to ver.22. Everything went ok.
When I had ver.17, the album covers were not displayed, only the station logo. That's why this is my question, will ver 22 change it or do I need to add something there

**Comment by 44Cheetah** [comment]
> Hello
> 
> I have such a problem I uploaded MIB 3.5.0, I was able to install AA / CP and update navigation maps for 2022/2023. It was the map version P197_N60S5MIBH3_EU. When I try to update the maps for 2023/2024 P330_N60S5MIBH3_EU now, the system tells me that there is no data to update. Please help
> 
> [MU1197-JTB-07427.10.181191015A.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11413834/MU1197-JTB-07427.10.181191015A.log)

Hello brother i search firmware MHI2_ER_BYG24_P4220 dl you have it ?

**Reply by Mriusz69** [reply]
Hello
I tried to make a full backup of the system but after 1.5 hours I stopped because the end of the backup was not yet in sight.
It took too long.
Regards

**Reply by 44Cheetah** [reply]
Hi ,
Oh no ! :( i need it why is so long ?

**Reply by 44Cheetah** [reply]
> Hello I tried to make a full backup of the system but after 1.5 hours I stopped because the end of the backup was not yet in sight. It took too long. Regards

Hi 
How i can get it ? 😔
I need him 
Thanks a lot 
Regards

**Reply by Mriusz69** [reply]
Sorry but I won't waste my time making a full backup. It's taking too long.
Regards

**Reply by 44Cheetah** [reply]
Is not stock in sdcard ?

**Reply by Mriusz69** [reply]
The old copy is gone, the card has been formatted.

**Reply by 44Cheetah** [reply]
Oh no !!!:( its so difficult to get the firmware very  very difficult,  you can make a backup in the night you put à charger in the car and syart the backup ?
Of course if you have garage or secure house .
